docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -138,7 +139,7 @@ In the event of a total active region loss, the following data will be lost:
 - Zeebe cluster scaling is not supported.
 - Web-Modeler is a standalone component and is not covered in this guide.
   - Modeling applications can operate independently outside of the automation clusters.
-- Kubernetes service meshes are currently unsupported, and we advise against their use for the setup.
+- We advise against using Kubernetes service meshes for the dual-region setup.
```

Comment:
at this point, what's the difference to just omitting it?

A customer can disregard it anyway and we haven't tested it to give a factual reasoning on why we advise against it. It results in more questions and potentially more support requests / forum posts on the topic than just not mentioning it at all.

Reply:
Good catch, fixing.